styles/utilities.css

```diff
@@ -2,6 +2,10 @@
   margin: 0 auto;
 }
 
+.mx-0 {
+  margin: 0;
```

Comment:
This does not set the `margin-left` and `margin-right`, but all margins. The name should then be `m-0` or you should implement it like this:
```css
.mx-0 {
  margin-left: 0;
  margin-right: 0;
}
```